CMakeLists.txt

```diff
@@ -128,6 +128,7 @@ add_library(libninja OBJECT
 	src/build_log.cc
 	src/build.cc
 	src/clean.cc
+	src/clsourcedependencies_parser.cc
```

Comment:
Shouldn't this depend on the value of `WIN32` since the check / fetch for rapidjson below is under that condition?

Reply:
Oh I see that there is an big `#ifdef NINJA_ENABLE_CL_SOURCE_DEPENDENCIES .. #endif` block in the source file. However some compilers will complain when trying to compile an empty file (after pre-processing). Can you make this addition conditional too then?

Reply:
I made addition of the source conditional and dropped whole-file `#if` conditions.